src/GafferUI/AnimationGadget.cpp

```diff
@@ -718,7 +717,7 @@ void AnimationGadget::renderLayer( Layer layer, const Style *style, RenderReason
 
 			glPushMatrix();
 
-			std::string label = boost::str( formatX % x.second );
+			const std::string label = fmt::format( "{:.2f}", x.second );
 			Box3f labelBound = style->textBound( Style::BodyText, label );
```

Comment:
It probably doesn't make any difference, but this feels like the sort of place where FMT_COMPILE would be quite reasonable.  Might be interesting sometime to benchmark ( and see if the compile time hit is measurable ).

Reply:
I'd be interested in the results of a `FMT_COMPILE` benchmark, but I think we can probably find a better place to do it than here.